src/Messages/AbstractResponse.php

```diff
@@ -6,8 +6,18 @@
 
 abstract class AbstractResponse extends CommonAbstractResponse
 {
+    /**
+     * Legacy method to maintain backwards compatibility
+     *
+     * @return mixed
+     */
     public function getTransactionReference()
     {
-        return $this->data['VK_REF'] ?? $this->data['VK_REF'];
+        return $this->getTransactionId();
+    }
+
+    public function getTransactionId()
+    {
+        return $this->data['VK_STAMP'] ?? $this->data['VK_STAMP'];
```

Comment:
Kapēc `null` gadījumā tiek atgriezts tas pats parametrs ? 
Vai pareizāk nebūtu `return $this->data['VK_STAMP'];` ?